Platform/Shared/VMRemovableDrivesView.swift

```diff
@@ -18,7 +18,7 @@ import SwiftUI
 
 @available(iOS 14, macOS 11, *)
 struct VMRemovableDrivesView: View {
-    let vm: UTMVirtualMachine
+    @State var vm: UTMVirtualMachine
```

Comment:
Is this needed in other places where I have `let vm` as well? My understanding is that as long as the final path is an ObservableObject we should be good? At least that would explain it working in other places...

Reply:
You're right, this is not needed. It is a remnant from debugging.